libs/platform/src/lib/components/approval-flow/approval-flow.component.html

```diff
@@ -0,0 +1,74 @@
+            {{ _maxCarouselStep - _carouselStep }}
+            <fd-icon [glyph]="'navigation-' + (_isRTL ? 'left' : 'right') + '-arrow'"></fd-icon>
+        </button>
+    </div>
+    <div class="approval-flow__graph-container" #graphContainerEl>
+        <div class="approval-flow__graph" #graphEl [style.left.px]="_isRTL ? -_carouselScrollX : _carouselScrollX" [attr.dir]="_dir">
+            <div class="approval-flow__column"
+                 *ngFor="let column of _graph; let columnIndex = index; let firstColumn = first; let lastColumn = last"
+                 [class.approval-flow__column--vertical-line-before]="column.nodes.length > 1 && !column.isPartial"
+                 [class.approval-flow__column--vertical-line-after]="column.isPartial || (column.nodes.length > 1 && _graph[columnIndex + 1]?.nodes.length === 1)"
+                 [class.approval-flow__column--vertical-line-after-solid]="column.allNodesApproved"
+            >
+                <ng-container
+                    *ngFor="let node of column.nodes; let nodeIndex = index; let firstNode = first; let lastNode = last">
+                    <fdp-approval-flow-node
+                        [attr.dir]="_dir"
+                        [attr.tabindex]="node.blank ? -1 : 0"
+                        [node]="node"
+                        [blank]="node.blank"
+                        [parent]="_nodeParentsMap[node.id]"
+                        [renderArrow]="columnIndex > 0"
+                        [renderLineBefore]="(_isCarousel && firstColumn && firstNode) || !!_nodeParentsMap[node.id] && !_graph[columnIndex - 1]?.nodes[nodeIndex]?.blank"
+                        [renderLineAfter]="(_isCarousel && lastColumn && lastNode) || !lastColumn && !node.blank && !_graph[columnIndex + 1]?.nodes[nodeIndex]?.blank"
+                        [renderCarouselStartMarker]="(_isCarousel && firstColumn && firstNode)"
+                        [renderCarouselEndMarker]="(_isCarousel && lastColumn && lastNode)"
+                        (onNodeClick)="onNodeClick(node)"
+                        (keyup.enter)="onNodeClick(node)"
+                        (keydown)="onNodeKeyDown($event, node, nodeIndex, columnIndex, firstColumn, firstNode, lastColumn, lastNode)"
+                    ></fdp-approval-flow-node>
+                </ng-container>
+            </div>
+        </div>
+    </div>
+</div>
+
+<ng-template let-messageToast #reminderTemplate>
+    <ng-container i18n="@@platformApprovalFlowReminderMessage"> Reminder has been sent to </ng-container>
+    <ng-container *ngIf="messageToast.data.targets.length === 1">
+        {{ messageToast.data.targets[0].name }}
+    </ng-container>
+    <ng-container *ngIf="messageToast.data.targets.length > 1">
+        {{ messageToast.data.targets.length }}
+        <ng-container i18n="@@platformApprovalFlowReminderMessageTeamMembersPrefix">members of</ng-container>
+        {{ messageToast.data.node.description }}
+    </ng-container>
+</ng-template>
```

Comment:
Looks like it's a good case for the `plural` rule ([i18n plural doc](https://angular.io/guide/i18n#mark-plurals)) so let's use it here. 
As result we should have one `i18n` marker for the whole message.
The same applicable for other similar cases.
The main idea is to keep translation unit meaningful and clear enough showing the whole context for a translator.

Reply:
thanks, changed to `plural`